tox.ini

```diff
@@ -341,7 +342,8 @@ deps =
     framework_tornado: pycurl
     framework_tornado-tornadolatest: tornado
     framework_tornado-tornadomaster: https://github.com/tornadoweb/tornado/archive/master.zip
-    mlmodel_openai: openai[datalib]<1.0
+    mlmodel_openai-openai0: openai[datalib]<1.0
+    mlmodel_openai-openai1: openai[datalib]<2.0
```

Comment:
We can leave this for now but we may want to also specifically test for the latest version of OpenAI to make sure we're alerted of future breaking changes.